unity-client/Assets/Scripts/MainScripts/DCL/Components/UUIDComponent/Tests/UUIDComponentTests.cs

```diff
@@ -887,7 +887,105 @@ public IEnumerator OnPointerDownEventAndPointerBlockerShape()
             Assert.IsTrue(targetEntityHit, "Target entity wasn't hit and no other entity is blocking it");
         }
 
+        [UnityTest]
+        public IEnumerator PointerEventNotTriggeredByParent()
+        {
+            Assert.IsNotNull(cameraController, "camera is null?");
+
+            // Create parent entity
+            DecentralandEntity blockingEntity;
+            BoxShape blockingShape;
+            InstantiateEntityWithShape(out blockingEntity, out blockingShape);
```

Comment:
We can make this code a bit more compact using the inline out declaration, like this:
```suggestion
            InstantiateEntityWithShape(out DecentralandEntity blockingEntity, out BoxShape blockingShape);
```